lib/Utils/Backends.pm

```diff
@@ -47,40 +59,84 @@ our %EXPORT_TAGS = (
 
 # Use it after SUT boot finish, as it requires ssh connection to SUT to
 # interact with SUT, including window and serial console
+
+=head2 use_ssh_serial_console
+
+Selects the root-ssh and saves it to SERIALDEV
+
+=cut
+
 sub use_ssh_serial_console {
     select_console('root-ssh');
     $serialdev = 'sshserial';
     set_var('SERIALDEV', $serialdev);
     bmwqemu::save_vars();
 }
 
+=head2 is_remote_backend
+
+remote backend
+
+=cut
+
 sub is_remote_backend {
     # s390x uses only remote repos
-    return check_var('ARCH', 's390x') || get_var('BACKEND', '') =~ /ipmi|spvm|svirt/;
+    return check_var('ARCH', 's390x') || (get_var('BACKEND', '') =~ /ipmi|svirt/) || is_spvm();
 }
 
 # In some cases we are using a VNC connection provided by the hypervisor that
 # allows access to the ttys same as for accessing any remote libvirt instance
 # but not what we use for s390x-kvm.
+
+=head2 has_ttys
+
+remote backend using ttys for: ipmi, s390x, spvm, except S390_ZKVM
```

Comment:
Please combine this with the brief comment that is above this method.

Reply:
updated, thanks